### DISCUSSION TITLE
GitHub Desktop Tutorial

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/data-science#16] [Learning Resources]
**YouTube tutorial: https://www.youtube.com/watch?v=8Dd7KRpKeaE&t=398s&ab_channel=CoderCoder** 👀 

Please comment below how help full this video was:
- give a rating out 5 for learning github desktop
- what stuck out and you learned
- what was confusing
- general thoughts

**Comment by AnaPeraltaG** [comment]
- Rating 4/5 (just because she used VS and I couldn't follow her since I was using R)
- How to create a repository using the desktop app; make changes in a document and push/pull/commit; create branches
- She uses VS code to create html and txt documents in the example. I'm not familiar with VS code but was able to create/modify documents and push/pull/commit using documents from R. Still, i found that a bit confusing at first. 
- Good video :) Not too long, very detailed; it's easy to follow her through.

**Reply by ran-codes** [reply]
thanks for the comments 😄

**Comment by AnaPeraltaG** [comment]
@ran-codes 

Forgot to add some questions:

- I created my GitHub account with my Drexel email address. Is it a good idea to create my account using a different email address or it doesn't really matter in the long term?
- When creating a new repository on Github, she mentioned that she usually selects "none" in the _Git ignore_ option. Did we select something different when we created the repository for my project? If so, what did we chose? 

Thank  you!!

**Reply by ran-codes** [reply]
## Email linked to GitHub

It's fairly easy to change your primary email linked to GitHub so don't worry about permenance as you can always change. What I recomend is to keep to the email that you use most frequently for work. so you ahve all your correspondence int eh same place (and you won't recieve work emails from GitHub to your non-work email 😄 )!

## Gitignore

You can't go wrong if you selection 'none' or another option. I believe we selected the 'R' specific option which creates ignore rules for Rstudio specific things!

The the key thing here is you can always edit the `.gitignore` file down the line to fit your project needs.
